### PR TITLE
Improvements to the "Compare builds" UI

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

File: frontend/static/templates/compare.html [hunk 1]
```diff
@@ -86,8 +86,8 @@
       </tr>
       <tr ng-hide='hide'>
         <th style="width: 60%">name (sample size)</th>
-        <th class="text-right">current (stdev)</th>
-        <th class="text-right">previous (stdev)</th>
+        <th class="text-right">#{{buildFrom.id}} (stdev)</th>
```
Comment (mwasilew) [inline frontend/static/templates/compare.html]: maybe add a link as well <a href='#/build/{{buildFrom.id}}'>#{{buildFrom.id}}</a> ?

**Comment by mwasilew** [comment]
I'm not sure I understand. Does this mean that the build details page won't show the comparison to baseline from now on? Did ART guys ask for that?

File: frontend/static/templates/compare.html (hunk 1 above)
Comment (terceiro) [inline frontend/static/templates/compare.html]: ok. will add links everywhere :)

**Comment by terceiro** [comment]
I would like to get rid of the comparison code in the build details, because it is now redundant, and has duplicated code. for example if we don't get rid of that, I will have to go back and also do all the small improvements to the benchmark comparison code there, *or* I would have to extract the duplicated code to e.g. a partial template so that we only have the code in one place.

**Comment by mwasilew** [comment]
I understand this approach and I'm perfectly OK with that. Let's talk to ART guys and see whether they're OK with removing the comparison from details page (than we go with this patch) or they're not (and we have to do subtemplates as you proposed).

**Comment by terceiro** [comment]
Julien said nobody had a strong opinion and that I could go ahead and do whatever it was best going forward, so I will merge this and deploy so I can go ahead with the other stuff.

**Pull request merged**